docs/source/markdown/podman-systemd.unit.5.md

```diff
@@ -27,7 +27,9 @@ corresponding regular systemd service unit files. Both system and user systemd u
 
 The Podman generator reads the search paths above and reads files with the extensions `.container`
 `.volume` and `*.kube`, and for each file generates a similarly named `.service` file. These units
-can be started and managed with systemctl like any other systemd service.
+can be started and managed with systemctl like any other systemd service.  Be aware that existing
```

Comment:
You have a redundant space after the period. Consider just starting a new line. In addition, maybe this comment should come one sentence earlier, just after we explain that a corresponding `.service` file will be created

Reply:
I've shuffled the text a bit.  The double space in between sentences is an old habit when writing monospace text.  I find it much easier to read.